modules/reissue-internal-certificates-central.adoc

```diff
@@ -7,21 +7,31 @@
 
 Central uses a built-in server certificate for authentication when communicating with other {product-title} services.
 This certificate is unique to your Central installation.
-The RHACS portal shows an information banner when the Central certificate is about to expire.
+The {product-title-short} portal shows an information banner when the Central certificate is about to expire.
 
 [NOTE]
 ====
-The information banner only appears 15 days before the certificate expiry date.
+The information banner only appears 15 days before the certificate expiration date.
 ====
 
+For {ocp} installations, beginning with {product-title-short} version 4.3.4, the Operator will automatically rotate all Central components' service TLS certificates 6 months before they expire. The following conditions apply:
+
+* The rotation of certificates in the secrets does not trigger the components to automatically reload them. However, reloads typically occur when the pod is replaced as part of an {product-title-short} upgrade or as a result of node reboots. If neither of those events happens at least every 6 months, you must restart the pods before the old (in-memory) service certificates expire. For example, you can delete the pods with an `app` label that contains one of the values of `central`, `central-db`, `scanner`, or `scanner-db`.
+
+* CA certificates are not updated. They are valid for 5 years.
+
+* The service certificates in the init bundles used by secured cluster components are not updated. You must rotate the init bundles at regular intervals.
+
+For Kubernetes installations, you must manually rotate TLS certificates. Instructions for manually rotating certificates are included in the following section.
```

Comment:
Same as above here. For installations not using the operator you have to manually rotate. No matter if its K8s/OCP